docs/mkdocs/docs/index.md

```diff
@@ -1,387 +1,495 @@
-![](images/FullWithBorder.png)
+ ![](images/FullWithBorder.png)
 
 ## What is ArcticDB?
 
-ArcticDB is an embedded/serverless database engine designed to integrate with Pandas and the Python Data Science ecosystem. ArcticDB enables 
-you to store, retrieve and process DataFrames at scale, backed by commodity object storage (S3-compatible storages and Azure Blob Storage).
+ArcticDB is a serverless DataFrame database engine designed for the Python Data Science ecosystem. 
+
+ArcticDB enables you to store, retrieve and process DataFrames at scale, backed by commodity object storage.
 
 ArcticDB requires *zero additional infrastructure* beyond a running Python environment and access to object storage and can be **installed in seconds.**
 
 ArcticDB is:
 
-- **Fast**: ArcticDB is incredibly fast, able to process millions of (on-disk) rows a second, and is very easy to install: `pip install arcticdb`!
-- **Flexible**: Supporting data with and without a schema, ArcticDB is also fully compatible with streaming data ingestion. The platform is bitemporal, allowing you to see all previous versions of stored data.
-- **Familiar**: ArcticDB is the world's simplest database, designed to be immediately familiar to anyone with prior Python and Pandas experience.
-
-#### What is ArcticDB _not_?
-
-ArcticDB is designed for high throughput analytical workloads. It is _not_ a transactional database and as such is not a replacement for tools such as PostgreSQL.
+- **Fast**
+    - Process millions of rows per second
+    - Quick and easy to install: `pip install arcticdb`
+- **Flexible**
+    - No data schema is needed
```

Comment:
This might be a bit misleading, as it's somewhat both right and wrong. As you note below append and update are not schemaless, but each write changes the schema. This requires us to start explaining dynamic vs static schema, which is advanced topic. I'm not sure we need it on the landing page. The landing page should be as small and concise as possible.

Reply:
That is covered further down